finagle-redis/src/main/scala/com/twitter/finagle/redis/protocol/commands/Sentinels.scala

```diff
@@ -0,0 +1,181 @@
+package com.twitter.finagle.redis.protocol.commands
+
+import com.twitter.finagle.redis.protocol._
+import com.twitter.finagle.redis.protocol.Commands._
+import com.twitter.finagle.redis.util._
+import com.twitter.finagle.redis.ClientError
+
+import java.net.InetSocketAddress
+
+import org.jboss.netty.buffer.ChannelBuffer
+import org.jboss.netty.buffer.ChannelBuffers
+
+case class SentinelMaster(name: ChannelBuffer)
+  extends Sentinel(SentinelMaster.channelBuffer, Seq(name))
+
+object SentinelMaster extends SentinelHelper {
+  val command = "MASTER"
+  def apply(args: Seq[Array[Byte]]): SentinelMaster = {
+    val name = trimList(args, 2, "SENTINEL MASTER")(0)
+    new SentinelMaster(ChannelBuffers.wrappedBuffer(name))
+  }
+}
+
+case class SentinelMasters() extends Sentinel(SentinelMasters.channelBuffer, Nil)
```

Comment:
case classes with no parameters were deprecated, so we should find some kind of solution to this–if it really bothers you to put them under the same object (I think it's fine) we can rename the object in this case.